packages/hardhat-core/src/internal/hardhat-network/provider/fork/ForkBlockchain.ts

```diff
@@ -222,6 +226,10 @@ export class ForkBlockchain implements HardhatBlockchainInterface {
     throw new Error("Method not implemented.");
   }
 
+  public getNetworkId() {
```

Comment:
I don't think this belongs in here. I'd rather have this info in the node. 

Also, we should add a `hardhat_getForkedNetworkId` or something like that.

Reply:
Not in this PR, but I mentioned it because that will lead to the Node having this info somehow.

Reply:
Changed in 1d866916